resources.tf

```
@@ -0,0 +1,315 @@
+#providers
+provider "aws" {
+	shared_credentials_file  = "C:\\Users\\ThulsiDossK\\.aws\\credentials"
+	region = "${var.region}"
+}
+
+#resources
+resource "aws_vpc" "vpc" {
+  cidr_block = "${var.cidr_vpc}"
+  enable_dns_support   = true
+  enable_dns_hostnames = true
+  tags = {
+    "Environment" = "${var.environment_tag}"
+  }
+}
```

- no need to comment on something that is obvious from the code itself
- `=` signs should be aligned across different lines
- `tags` section should be separated from the rest of the arguments using newlines
- Please apply the same recommendations to the rest of resources as well
```suggestion
resource "aws_vpc" "vpc" {
  cidr_block           = "${var.cidr_vpc}"
  enable_dns_support   = true
  enable_dns_hostnames = true
  
  tags = {
    "Environment" = "${var.environment_tag}"
  }
}
```